app/code/Magento/InventoryInStorePickupAdminUi/view/adminhtml/ui_component/inventory_source_form.xml

```diff
@@ -6,17 +6,17 @@
  */
 -->
 <form xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
-    <fieldset name="general" sortOrder="10">
-        <field name="in_store_pickup" formElement="checkbox" sortOrder="70">
+    <fieldset name="general">
+        <field name="is_pickup_location_active" formElement="checkbox" sortOrder="70">
             <argument name="data" xsi:type="array">
                 <item name="config" xsi:type="array">
-                    <item name="default" xsi:type="number">1</item>
+                    <item name="default" xsi:type="number">0</item>
                 </item>
             </argument>
             <settings>
-                <dataScope>extension_attributes.in_store_pickup</dataScope>
+                <dataScope>extension_attributes.is_pickup_location_active</dataScope>
                 <dataType>boolean</dataType>
-                <label translate="true">Supports In-Store Pickup</label>
+                <label translate="true">Eligible for in-store pickup</label>
```

Comment:
Do you think that development and debugging experience may be affected for someone looking for `Eligible for in-store pickup` but later finding that in the code it is `is_pickup_location_active`? 

I like `Eligible ...` wording more, however it's your call which one to use.

Reply:
I also like `Eligible`.
But to align names to one way, I have renamed to `Use as Pickup Location`.